packages/vite/src/node/preview.ts

```diff
@@ -72,6 +74,10 @@ export interface PreviewServerForHook {
    * Print server urls
    */
   printUrls(): void
+  /**
+   * Bind shortcuts
+   */
+  bindShortcuts(options?: BindShortcutsOptions): void
```

Comment:
```suggestion
  /**
   * Bind shortcuts
   */
  bindShortcuts(options?: BindShortcutsOptions<PreviewServer>): void
```

And here too. There's some other places that uses `BindShortcutsOptions` directly (e.g. the docs) that need a similar change.